msi/GitProduct.wxs

```diff
@@ -158,6 +167,11 @@
       <Directory Id='ProgramFiles$(var.SixtyFourBit)Folder' Name='PFiles'>
         <Directory Id='INSTALLFOLDER' Name='Git'>
           <Directory Id='BinFolder' Name='bin' />
+        <?if $(var.SixtyFourBit)=64 ?>
+          <Directory Id='MingwFolder' Name='mingw64' />
+        <?else?>
+          <Directory Id='MingwFolder' Name='mingw32' />
+        <?endif?>
```

Comment:
I _could_, but then the declaration of the directory structure would be a bit fragmented (it's a matter of stylistic opinion, I think):

```
# New line in the here-file generating GitComponents.wxs...
<Directory Id="MingwFolder" Name="mingw$BITNESS" />
```

But then, in `GitProduct.wxs` I would need to reference this as follows:

```
<!-- In GitProduct.wxs -->
...
<Directory Id='INSTALLFOLDER' Name='Git'>
    ...
    <DirectoryRef Id='MingwFolder' />
</Directory>
```

Neither way is more correct than the other. Which do you prefer?